software/base/src/main/java/brooklyn/entity/software/MachineLifecycleEffectorTasks.java

```diff
@@ -614,6 +631,23 @@ public void stop(ConfigBag parameters) {
         if (log.isDebugEnabled()) log.debug("Stopped software process entity "+entity());
     }
 
+    @VisibleForTesting
+    public static boolean canStop(StopMode stopMode, boolean isEntityStopped) {
```

Comment:
The method is used in a test from a different package, that's why the public visibility is needed.

I changed the code a bit - the method is now protected with the test code providing a public wrapper, no need for the annotation. wdyt?